tests/lib/Microsoft.Fx.Portability.Cci.Tests/TestAssembly.cs

```diff
@@ -17,7 +17,15 @@ namespace Microsoft.Fx.Portability.Cci.Tests
     internal class TestAssembly
     {
         private const string TFM = @"[assembly: global::System.Runtime.Versioning.TargetFrameworkAttribute("".NETFramework,Version=v4.5.1"", FrameworkDisplayName = "".NET Framework 4.5.1"")]";
-        private static readonly string Mscorlib = typeof(object).GetTypeInfo().Assembly.Location;
+        private static readonly IEnumerable<string> DefaultReferences = new[]
+        {
+            typeof(object).Assembly.Location,
+            typeof(Console).Assembly.Location,
+
+            // Reference to System.Runtime.dll rather than System.Private.CoreLib on .NET Core
+            typeof(RuntimeReflectionExtensions).Assembly.Location
+        }.Distinct();
```

Comment:
This is a fragile way to do this. The location of types isn't set, particularly in Core. If we want to be explicit about references, we should just build the explicit list.

Reply:
I could be persuaded that this is a nice to have.

Reply:
Created a new issue to track the concern, but don't block the integration.